pom.xml

```diff
@@ -44,7 +44,7 @@
         <postgresql.version>42.2.5</postgresql.version>
         <mockito.version>2.26.0</mockito.version>
         <cwlavro.version>2.0.4.2</cwlavro.version>
-        <okhttp.version>3.9.0</okhttp.version>
```

Comment:
This is more of a general comment, but noticed here that the powermock dependency isn't being deleted. 
Is it being used anywhere else?

Reply:
MockedIT

Reply:
Sad.
Guess we'll need a follow-up ticket, but later.